enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1186 @@
+supporting a longer term separation of control-plane and worker-node updates
+for **Standalone** cluster profiles into distinct concepts and phases of managing 
+an OpenShift cluster (HCP clusters already provide this distinction). The motivations
+for both aspects will be covered, but a special focus will be made on the motivation
+for separating Standalone control-plane and worker-node updates as, while not fully realized 
+by this enhancement alone, ultimately provides additional business value helping to 
+justify an investment in the new operational tool.
+
+### Supporting the Eventual Separation of Control-Plane and Worker-Node Updates
+One of the key value propositions of this proposal pre-supposes a successful
+decomposition of the existing, fully self-managed, Standalone update process into two  
+distinct phases as understood and controlled by the end-user: 
+(1) control-plane update and (2) worker-node updates.
+
+To some extent, Maintenance Schedules (a key supported option for change management) 
+are a solution to a problem that will be created by this separation: there is a perception that it would also
+double the operational burden for users updating a cluster (i.e. they have
+two phases to initiate and monitor instead of just one). In short, implementing the
+Maintenance Schedules concept allows users to succinctly express if and how
+they wish to differentiate these phases.
+
+Users well served by the fully self-managed update experience can disable 
+change management (i.e. not set an enforced maintenance schedule), specifying
+that control-plane and worker node updates can take place at
+any time. Users who need more control may choose to update their control-plane
+regularly (e.g. to patch CVEs) with a permissive change management configuration
+for the control-plane while using a tight maintenance schedule for worker-nodes
+to only update during specific, low utilization, periods.
+
+Since separating the node update phases is such an important driver for
+Maintenance Schedules, their motivations are heavily intertwined. The remainder of this 
+section, therefore, delves into the motivation for this separation. 
+
+#### The Case for Control-Plane and Worker-Node Separation
+From an overall platform perspective, we believe it is important to drive a distinction
+between updates of the control-plane and worker-nodes. Currently, an update is initiated
+and OpenShift's ostensibly fully self-managed update mechanics take over (CVO laying
+out new manifests, cluster operators rolling out new operands, etc.) culminating with 
+worker-nodes being drained a rebooted by the machine-config-operator (MCO) to align
+them with the version of OpenShift running on the control-plane.
+
+This approach has proven extraordinarily successful in providing a fast and reliable 
+control-plane update, but, in rare cases, the highly opinionated update process leads
+to less than ideal outcomes. 

```

Comment:
Should we re frame this section to combine more of what is in "Node Update Separation to Address Problems in User Perception" as the case here is more to give customers control on when to update worker nodes due to possible interruptions to their workloads.